client/a8c-for-agencies/sections/marketplace/lib/product-brand.tsx

```diff
@@ -0,0 +1,11 @@
+import {
+	PRODUCT_BRAND_FILTER_ALL,
+	PRODUCT_BRAND_FILTER_JETPACK,
+	PRODUCT_BRAND_FILTER_WOOCOMMERCE,
+} from '../constants';
+
+export function getValidBrand( brand: string ) {
+	return brand === PRODUCT_BRAND_FILTER_JETPACK || brand === PRODUCT_BRAND_FILTER_WOOCOMMERCE
+		? brand
+		: PRODUCT_BRAND_FILTER_ALL;
```

Comment:
What about creating an array of valid brands to keep it slightly cleaner when we add more brands soon?

```suggestion
	const validBrands = [ PRODUCT_BRAND_FILTER_JETPACK,  PRODUCT_BRAND_FILTER_WOOCOMMERCE ];
	
	return validBrands.includes( brand ) ? brand
		: PRODUCT_BRAND_FILTER_ALL;
```

Reply:
Good point. Thanks for the suggestion. This has been applied 👍

Reply:
Great!